### PR TITLE
OBPIH-7901 Check page logic (fix zeros and delivery date)

### PR DESCRIPTION
**Pull request by alannadolny**
pr related to: https://pihemr.atlassian.net/browse/OBPIH-7901?focusedCommentId=195762

### CODE REVIEW

File: src/test/groovy/org/pih/warehouse/api/receiving/v2/ReceiptV2ServiceSpec.groovy [hunk 1]
```diff
@@ -813,6 +813,20 @@ class ReceiptV2ServiceSpec extends Specification implements ServiceUnitTest<Rece
         1 * shipmentService.createShipmentEvent(shipment, _, EventCode.RECEIVED, shipment.destination)
     }
 
+    void 'completeReceipt should write out a zero on the lines that were never given a quantity'() {
```
Comment (ewaterman) [inline src/test/groovy/org/pih/warehouse/api/receiving/v2/ReceiptV2ServiceSpec.groovy:870]: can you also add a test for the case where partial receiving is disabled? Leaving a qty null should cancel all of it and then also set qty received to 0. See the above test for reference

**Review by ewaterman — APPROVED** [review]
approving but please add that other test case

**Review by SebastianLib — APPROVED** [review]

**Review by SebastianLib — APPROVED** [review]
